### ISSUE TITLE
CMake add_plugintest corrections

### ISSUE THREAD

**Issue by ingwinlu** [ElektraInitiative/libelektra#1995]
I stumbled upon the following comments (which are spread in at least 2 plugins):
```
# test environment clears env so setting LUA_CPATH is no option
# we workaround this by changing the current directory to our bindings
# output directory + per default lua loads modules from current directory
```

This is no longer true.

**Comment by markus2330** [comment]
Thank you for bringing this up! Can you please correct/remove the outdated information?

Btw. lua still fails for me when I run it from an installed Elektra. Maybe we should have an extra status for exclusion of plugins from the build server (like needsfix) and apply this to the lua plugin?

**Comment by markus2330** [comment]
Removing 3 lines of the docu should be a doable issue :wink:

**Comment by a-kraschitzer** [comment]
I would like to take this over.

**Comment by mpranj** [comment]
Closed via #3790 (origin: #3775) thanks to @a-kraschitzer!
